starbucks_dw/models/reporting/customer_offer_responses.sql

```diff
@@ -0,0 +1,45 @@
+WITH offer_details AS (
```

Comment:
You forgot to add `config` for each reporting model, which includes the absence of materialisation and database schema. Not setting it, you will have `view` as a materialisation and your reporting models will be on `public` schema.

Both of configuration can be done in `dbt_project.yml` or `config` block.